whatlies/embedding.py

```diff
@@ -93,6 +93,25 @@ def __sub__(self, other):
         copied.vector = self.vector - other.vector
         return copied
 
+    def __neg__(self):
+        """
+        Negate a embedding.
+
+        Usage:
+
+        ```python
+        from whatlies.embedding import Embedding
+
+        foo = Embedding("foo", [0.1, 0.3])
+
+        assert (- foo).vector == - foo.vector
+        ```
+        """
+        copied = deepcopy(self)
+        copied.name = f"(- {self.name})"
```

Comment:
I am not sure if the space between the negation and the name should be there or not. In math writing, we don't put it but here the situation might be a bit different.